pkg/console/subresource/deployment/deployment.go

```diff
@@ -77,13 +82,23 @@ func DefaultDeployment(operatorConfig *operatorv1.Console, cm *corev1.ConfigMap,
 		secretResourceVersionAnnotation:                      sec.GetResourceVersion(),
 		consoleImageAnnotation:                               util.GetImageEnv("CONSOLE_IMAGE"),
 	}
+	podAnnotations := deployment.Spec.Template.ObjectMeta.Annotations
+	for k, v := range deployment.ObjectMeta.Annotations {
+		podAnnotations[k] = v
+	}
+	deployment.Spec.Template.ObjectMeta.Annotations = podAnnotations
+}
 
-	// Set any annotations as needed so that `ApplyDeployment` rolls out a
-	// new version when they change.
-	meta.Annotations = deploymentAnnotations
+func withReplicas(deployment *appsv1.Deployment, infrastructureConfig *configv1.Infrastructure) {
```

Comment:
So I though that this will either replace the `Replicas()` function, since it doesnt make sense to have a function for determining the proper value(replica count in this set) and its wrapper thats being a setter. We should combine it into a single function. Also this way we would just keep enlarge our unit test cases without any real added value.

Reply:
@jhadvig done!